core/main/src/Core/Canvas.ts

```diff
@@ -487,6 +488,27 @@ export class Canvas {
                 particle.particlesOptions.shadow
             );
         }
+
+        const orbitOptions = particle.particlesOptions.orbit;
+        if (orbitOptions.enable === true) {
+            this.drawOrbit(particle, orbitOptions);
+        }
+    }
+
+    public drawOrbit(particle: IParticle, orbitOptions: IOrbit): void {
+        if (!this.context) {
+            return;
+        }
+
+        CanvasUtils.drawEllipse(
+            this.context,
+            particle,
+            orbitOptions.color || particle.getFillColor(),
+            typeof particle.orbitRadiusValue === "undefined" ? particle.getRadius() : particle.orbitRadiusValue,
```

Comment:
you can shorten this with the null check:

```javascript
particle.orbitRadiusValue ?? particle.getRadius()
```

Reply:
This was the previous code, but when setting radius as 0, you will end up not using 0 and use the particle's radius instead as its falsey I believe.

Reply:
Probably you were using the `||` operator which uses all falsy values, the `??` operator works only with `undefined` and `null`.

Reply:
Ok, that's cool. Didn't know that. thanks!